index.js

```diff
@@ -2,16 +2,35 @@
 const path = require('path');
 const eslint = require('eslint');
 const globby = require('globby');
+const multimatch = require('multimatch');
 const optionsManager = require('./options-manager');
 
 exports.lintText = (str, opts) => {
 	opts = optionsManager.preprocess(opts);
 
+	if (opts.cwd && opts.filename) {
```

Comment:
To be user-friendly, I think we should throw an error if the user of `lintText` tries to specify the `ignore` option, but didn't supply `cwd` and `filename`.